packages/angular-sdk-overrides/README.md

```diff
@@ -0,0 +1,3 @@
+# @pega/angular-sdk-overrides
+
+This Node package comprises templates for standalone Angular components that are designed to be overridden by the Angular SDK.
```

Comment:
Shouldn't we get rid of `templates` word here? We can rather say `This Node package comprises standalone Angular components that are designed to be overridden by the Angular SDK`?